### PR TITLE
Adding geolib definition with fixed signatures in file taken from Vladimir Venegas. Tests are also added.

### PR DESCRIPTION
**Pull request by DotNetPart**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: geolib/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictNullChecks": false,
```
Comment (sheetalkamat) [inline geolib/tsconfig.json]: strictNullChecks is required to be true in all new packages

Comment (sheetalkamat) [inline geolib/tsconfig.json]: strictNullChecks is required to be true in all new packages

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
Leave a comment

File: geolib/tsconfig.json (hunk 1 above)
Comment (DotNetPart) [inline geolib/tsconfig.json]: Done.

**Review by sheetalkamat — APPROVED** [review]

**Pull request merged**